msteams-platform/sbs-gs-notificationbot.yml

```diff
@@ -0,0 +1,465 @@
+
+    1. Select the location for project workspace.
+    1. Select **Select Folder**.
+
+        :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/select-folder.png" alt-text="Select Folder" border="true":::
+    
+    1. Enter a suitable name for your app, such as mynotificationbot as the application name. 
+    1. Select **Enter**.
+
+        :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/type-notfication-name.png" alt-text="Screenshot showing where to enter the app name." border="true":::
+
+       > [!NOTE]
+       > Ensure that you use only alphanumeric characters in the application name.
+
+        The notification bot app is created in a few seconds.
+
+        :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/my-notification-bot.png" alt-text="Screenshot showing the app created." border="true":::
+  
+    1. Select **Run and Debug** :::image type="icon" source="../msteams-platform/assets/images/teams-toolkit-v2/run-debug-icon.png"::: icon from the Visual Studio Code sidebar.
+
+    1. Select **Debug(Edge)** or **Debug(Chrome)** from the dropdown list.
+
+    1. Select **Start Debugging** button.
+
+        :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/debug-app.png" alt-text="App Debugging"border="true"::: 
+
+    1. Select **Add** 
+
+        :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/my-notification-output.png" alt-text="My Notification Output1"border="true":::
+
+    1. Open **Windows PowerShell**.
+
+    1. Run the `Invoke-Webrequest -Method POST -URI http://localhost:3978/api/notification` command.
+
+         :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/windows-powershell.png" alt-text="Powershell Trigger"border="true":::   
+
+        You'll get the following output in Teams:
+
+          :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/trigger-output.png" alt-text="Trigger Output"border="true":::
+
+    ## Take tour of source code
+
+    Teams Toolkit provides components for building an app. After creating the project, you can view the project folders and files in the **Explorer** area of Visual Studio Code.
+
+    :::image type="content" source="~/assets/images/sbs-notification-bot/folder-structure.png" alt-text="structure tab" border="true":::
+
+    If you choose to use UI framework, following sample template code provides a scaffolding with React components.
+
+    The new project folder contains following content:
+
+    | Folder name | Contents |
+    | --- | --- |
+    | `.fx` | Project level settings, configuration, and environment information |
+    | `.vs code` | VS code files for local debug |
```

Comment:
There is no space between .vs and code, it should be ".vscode".

Reply:
Updated as mentioned